### PR TITLE
ISPN-5956 Wait for the test cache to rebalance

### PR DESCRIPTION
**Pull request by danberindei**
https://issues.jboss.org/browse/ISPN-5956

### CODE REVIEW

**Comment by pruivo** [comment]
unused ```import org.infinispan.test.TestingUtil;``` in almost all of the changed files

File: core/src/test/java/org/infinispan/partitionhandling/BaseTxPartitionAndMergeTest.java [hunk 1]
```diff
@@ -80,9 +82,10 @@ protected FilterCollection createFilters(String cacheName, boolean discard, Clas
 
    protected abstract Log getLog();
 
-   protected void mergeCluster() {
+   protected void mergeCluster(String cacheName) {
       getLog().debugf("Merging cluster");
       partition(0).merge(partition(1));
+      TestingUtil.waitForRehashToComplete(caches(cacheName));
```
Comment (pruivo) [inline core/src/test/java/org/infinispan/partitionhandling/BaseTxPartitionAndMergeTest.java]: you can remove the ```TestingUtil.``` prefix and the import

**Comment by danberindei** [comment]
I removed the imports.

**Comment by pruivo** [comment]
looks good. pushing...

**Comment by pruivo** [comment]
integrated! thanks @danberindei !

**Pull request closed without merging**